anchore_engine/db/entities/upgrade.py

```diff
@@ -1955,6 +1960,52 @@ def db_upgrade_013_014():
     upgrade_oauth_client_014()
 
 
+def remove_incorrect_github_vuln_data():
+    """
+    Github feed data was being processed under incorrect types 'github:os' and 'github:unknown'. This function removes
+    those vulnerability records synced by the policy engine because they are incorrect data
+    """
+    engine = anchore_engine.db.entities.common.get_engine()
+
+    namespace_names = ["github:os", "github:unknown"]
+
+    to_delete = (
+        (
+            ImagePackageVulnerability,
+            ImagePackageVulnerability.vulnerability_namespace_name,
+        ),
+        (VulnerableArtifact, VulnerableArtifact.namespace_name),
+        (FixedArtifact, FixedArtifact.namespace_name),
```

Comment:
(nit) VulnerableArtifact and FixedArtifact deletion is not necessary. Vulnerability is related to those entities via a 1-many relationship. The [foreign key relationship](https://github.com/anchore/anchore-engine/blob/master/anchore_engine/db/entities/policy_engine.py#L287-L292) also defines cascade operations. This would mean that deleting the parent Vulnerability record would delete all the related children in both tables. The end result is same regardless with a minor/insignificant performance gain as the data set is very small. IMO delegating cascade deletion to ORM is more precise and reduces the possibility of orphaning records.